### PR TITLE
documents premadepapers.dm

### PR DESCRIPTION
**Pull request by ReddicusDragon**
fuck autodoc

literally a test of autodocumentation since I'm a shitcoder

:cl:  
rscadd: added documentation to paperwork
/:cl:

### CODE REVIEW

File: yogstation/code/game/objects/items/premadepapers.dm [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
-//fucking shitcode
-//REALLY BAD SHITCODE HOLY FUCK
-
-
+/**
+  *
+  *
+  * A General Request Form for things without a form
+  *
+  * That's literally it.
```
Comment (swissloaf) [inline yogstation/code/game/objects/items/premadepapers.dm:6]: "That's literally all it is." dont think is necessary in every paper tbh

**Review by swissloaf — CHANGES_REQUESTED** [review]

**Comment by MrHorizons** [comment]
Since they're all single-line, couldn't you just use `///` instead of using all the multi-line markup?

**Comment by ReddicusDragon** [comment]
No bc autodoc

**Comment by MrHorizons** [comment]
... I thought that was autodoc markup, but okay.

**Comment by ReddicusDragon** [comment]
@MrHorizons wdym? Based off the guide, the multiline one is used for things that aren't just variables

**Review by alexkar598 — APPROVED** [review]

**Pull request merged**